test/helpers/utils.go

```diff
@@ -558,6 +558,24 @@ func DoesNotSupportMetalLB() bool {
 	return true
 }
 
+func (kub *Kubectl) HasHostReachableServices(pod string, checkTCP, checkUDP bool) bool {
+	status := kub.CiliumExecContext(context.TODO(), pod,
```

Comment:
Not a fan of using `context.TODO()`, but given how nested this func is, I it would be detrimental to readability to pass newly created context from the test case.